.github/workflows/stale.yaml

```diff
@@ -0,0 +1,15 @@
+name: 'Automatically close stale PRs'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: action/stale@v8
+        with:
+          start-date: '2023-05-01T00:00:00Z'
+          stale-pr-message: 'This PR has been opened without with no activity for 180 days. Comment on the issue otherwise it will be closed in 7 days'
```

Comment:
An education question for me rather than a possible problem, but is this fact we only have `stale-pr-message` and not `stale-issue-message` what makes this just run on PRs and we don't need to set [`days-before-pr-stale`](https://github.com/actions/stale#days-before-pr-stale) instead of `days-before-stale`?

Reply:
No, I think I explicitly will have to change the issues to -1 and make sure those are not closed for now. :-)
Thanks for catching this, I will update it soon.

Reply:
Updated!